frontend/src/config/types.ts

```diff
@@ -365,11 +365,13 @@ export enum LabelType {
   Date = 'date',
   Text = 'text',
   Number = 'number',
+  ValuesLabels = 'values_labels',
 }
 
 interface FeatureInfoProps {
   type: LabelType;
   label: string;
```

Comment:
I think that the current parameter names and types are confusing. Between `value, values, label, ValuesLabels` it's easy to get lost.

Can we find better names?